sources/osgViewer/input/source/InputSourceWebVR.js

```diff
@@ -20,9 +25,10 @@ var InputSourceWebVR = function() {
     }
     this._nbCallbacks = 0;
     // 3 seconds default poll interval
-    this._pollInterval = 3000;
+    this._pollInterval =
+        options && options.pollInterval !== undefined ? options.pollInterval : 3000;
```

Comment:
3000 is quite big ?
maybe a comment explaining the choice

Reply:
3 seconds seemed an ok value. It's the maximum time a user will have to wait when pluging his headset to be recognized.

Reply:
ha ok, it's the time between putting an headset and getting result ?

When does that arrives ?

once user clicks on the "vr icon" we poll each 3 secs ? or it's at another point ?
(or maybe it's always running ?)